### PR TITLE
add stevenctl to org

### PR DESCRIPTION
**Pull request by stevenctl**
If you are joining the org, please fill out the template below:

- [X] I have reviewed the [community membership guidelines](https://github.com/istio/community/blob/master/ROLES.md#member)
- [X] I have reviewed the [contribution guidelines](https://github.com/istio/community/blob/master/CONTRIBUTING.md)
- [X] I have enabled [2FA on my GitHub account](https://github.com/settings/security)
- [X] I have joined the [Istio discussion board](https://discuss.istio.io) and subscribed to the [Contributors](https://discuss.istio.io/c/contributors) category
- [x] I have joined [Istio's Slack workspace](https://istio.slack.com) (fill-out this [form](https://docs.google.com/forms/d/e/1FAIpQLSfdsupDfOWBtNVvVvXED6ULxtR4UIsYGCH_cQcRr0VcG1ZqQQ/viewform) to join)

List your company name, or indicate Individual if you're not affiliated with a company: Google
Provide a link to at least one PR that you have successfully pushed to one
of the Istio repos: https://github.com/istio/istio/pull/22946

### CODE REVIEW

**Comment by stevenctl** [comment]
@duderino

File: org/istio.yaml [hunk 1]
```diff
@@ -501,6 +503,7 @@ orgs:
         - SataQiu
         - sbezverk
         - sdake
+        - stevenctl
```
Comment (duderino) [inline org/istio.yaml:506]: @stevenctl sorry, I gave you bad advice.  Could you remove this line for now?  You should sign up to be a member, but not yet a maintainer, though I'm sure that'll change quickly

File: org/istio.yaml [hunk 2]
```diff
@@ -754,6 +757,7 @@ orgs:
             - ramaraochavali
             - rlenglet
             - sbezverk
+            - stevenctl
```
Comment (duderino) [inline org/istio.yaml:760]: Also please remove this line as well

Comment (duderino) [inline org/istio.yaml:760]:
```suggestion
```

File: org/istio.yaml (hunk 1 above)
Comment (duderino) [inline org/istio.yaml:506]:
```suggestion
```

**Pull request merged**